README.rst

```diff
@@ -78,3 +99,12 @@ Next Steps
 - and execute the setup.py file like , python setup.py install.
 
 - mostly if you change you imports from from google.cloud to clearblade.cloud everything else should work.
+
+Note about types of times and binaryData
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- By default time parameters (e.g. **cloudUpdateTime**, **deviceAckTime**, **updateTime**) are returned as **RFC3339** strings (e.g. "2023-01-12T23:38:07.732Z").
+- To return times formatted as **DatetimeWithNanoseconds** (defined in the **google.api_core.datetime_helpers** module) as returned by the **Google IoTCore Python SDK**, set environment variable **TIME_FORMAT** to exactly **datetimewithnanoseconds**.
```

Comment:
I thought about that. As we observed it seems all env. vars have to be strings. So we would have to account for "true" vs "True" vs "TRUE". That is not a big deal: I'm already converting the passed env. var to lower case anyway.

With the time format, the thing word LEGACY would confuse me: it's not like we supported that type before and we are now transitioning away from it.

And with ALLOW_BYTE_ARRAYS I would wonder where are we now allowing BYTE_ARRAYS that we didn't before? Setting that env. var ONLY changes **binaryData** and it changes EVERY instance of **binaryData**. Which is why BINARYDATA_FORMAT makes sense to me.

Reply:
> Should we just change the env variables values to true (false being the default). `datetimewithnanoseconds` value might be a bit too long and people might mess up writing that. We can ask Aaron/Ronak for their opinion on this.

Yeah I agree with that. If someone wrote wrong spelling it will also cause the issue and true false makes more sense.

Reply:
But I'm fine with that - just couldn't come up with variable names that were not confusing.